src/js/components/TabPanesComponent.jsx

```diff
@@ -107,6 +116,18 @@ var TabPanesComponent = React.createClass({
       viewType: AppListViewTypes.LIST
     };
 
+    var filterStatusClearLink = state.filterStatus.length > 0
+      ? <a onClick={this.handleClearFilter.bind(null, "filterStatus")}>Clear</a>
```

Comment:
Let's use ```Link``` here to be consistent with the react router and avoid setting click-handler on an a-element.
Instead of ```handleClearFilter``` you could use something like ```getQueryParamsWithout``` to set the existing params.